test/cluster/dtest/audit_test.py

```diff
@@ -613,7 +613,7 @@ def execute_and_validate_audit_entry(query, category, **kwargs):
             return self.execute_and_validate_audit_entry(session, query, category, audit_settings, **kwargs)
 
         execute_and_validate_audit_entry(
-            "CREATE KEYSPACE ks WITH replication = { 'class':'SimpleStrategy', 'replication_factor':1} AND DURABLE_WRITES = true",
+            "CREATE KEYSPACE ks WITH replication = { 'class':'SimpleStrategy', 'replication_factor':1} AND tablets = {'enabled': false} AND DURABLE_WRITES = true",
```

Comment:
Maybe it's out of scope of this PR, but why stick to vnodes here rather than using NTS and letting tablets be used?